### ISSUE TITLE
Kismatic should use a standard policy of labels/taints to avoid scheduling on master

### ISSUE THREAD

**Issue by truncj** [apprenda/kismatic#562]
Other configuration tools such as kubeadm use a label "node-role.kubernetes.io/master=" to identify the master so that pods can be specified using node-selectors. Pods with these node-selectors will fail by default on Kismatic clusters due to the master being "cordoned". A more straightforward way of avoiding scheduling would be the use of standardized taints/tolerations so the user can see why this behavior is occurring.

**Comment by alexbrand** [comment]
Discussion around this (including that specific label) has been ongoing for some time now: https://github.com/kubernetes/kubernetes/pull/39112. I think we can implement this once there's consensus, but from what I can see it doesn't look like we there's a "standard" set of labels or taints/tolerations quite yet.

**Comment by truncj** [comment]
Fair enough, thanks! Just running into these issues while trying to setup Contiv on a non-kubeadm setup.

**Comment by swade1987** [comment]
As kubernetes/kubernetes#39112 has closed we should probably align to what was agreed.
